src/modules/colorPicker/ColorPickerUI/Controls/ColorFormatControl.xaml

```diff
@@ -49,11 +49,25 @@
                     Width="36"
                     Grid.Column="2"
                     AutomationProperties.HelpText="{Binding ElementName=FormatNameTextBlock, Path=Text}"
-                    AutomationProperties.Name="{x:Static p:Resources.Copy_to_clipboard}"
-                    FontSize="12"
-                    Style="{StaticResource DefaultButtonStyle}"
-                    FontFamily="Segoe MDL2 Assets"
-                    Content="&#xE16F;" />       
+
+                    AutomationProperties.Name="{x:Static p:Resources.Copy_to_clipboard}">
+                <Button.Content>
+                    <TextBlock FontFamily="Segoe MDL2 Assets"
+                           Text="&#xE8C8;"
+                           FontSize="12" />
+                </Button.Content>
+                <Button.Resources>
```

Comment:
why was this added?

Reply:
You mean the Textbock? I have no idea, I honestly did my best to focus on just the unicode points.

Reply:
The resource / trigger.  It wasn't there before.

Same with the TextBlock.

Reply:
Can't remember I typed these things.
If this pr is not good enough im going to ask @niels9001 to take it over. Maybe with a new pr. You probably do this in 5 minutes or so. In that case: maybe you could also try to find other filetypes that use the other unicode format?